recipe/meta.yaml

```diff
@@ -58,9 +61,11 @@ requirements:
         - libpng 1.6*
         - zlib 1.2*  # [not win]
         - jpeg 9*
-        - postgresql  # [linux]
+        - postgresql  # [not win]
         - openjpeg
         - curl
+        - sqlite 
```

Comment:
We can try to modify the `bld.bat` to use this on Windows too, but for now I would like to add the `#  [not win]` here too to avoid giving people the impression that we are using `sqlite` on Windows.